content/10-minute-tutorial.md

```diff
@@ -110,12 +146,102 @@ You should see something like the following:
 
 Your project builds correctly, but nothing is tested yet as you have not specified any behaviour to test against.
 
+{{% /block %}}
+
+{{% block "javascript" %}}
+
+Create a new working directory called `cucumber-js` and change into it. The rest of tutorial assumes that you are in this directory.
```

Comment:
I would use a different project name instead of having the same name as the cucumber-js report. Maybe `my-project`